Sources/Purchasing/StoreKit2/StoreKit2StorefrontListener.swift

```diff
@@ -32,17 +32,29 @@ class StoreKit2StorefrontListener {
     }
 
     weak var delegate: StoreKit2StorefrontListenerDelegate?
+    private let updates: AsyncStream<StorefrontType>
 
-    init(delegate: StoreKit2StorefrontListenerDelegate?) {
+    convenience init(delegate: StoreKit2StorefrontListenerDelegate?) {
+        self.init(
+            delegate: delegate,
+            updates: StoreKit.Storefront.updates.map(Storefront.init(sk2Storefront:))
+        )
+    }
+
+    init<S: AsyncSequence>(
```

Comment:
let's add a description and sample usage here, this `init` won't be intuitive to folks not very used to Swift

Reply:
I added a comment explaining this.